### PR TITLE
Update symbols for build information (View/Edit)

### PR DESCRIPTION
**Pull request by DuMaM**
Following with suggestion of @uhafner i'm adding ionic symbol for edit-note and open-note ([here](https://github.com/jenkinsci/workflow-job-plugin/pull/274#pullrequestreview-1031619607))
Those icons replace symbol gear which doesn't describe too well what is going on here.
Gear symbol is also a bit overused across whole Jenkins and plugins, what is another reason for this PR.

Those icons was provided from this site: https://ionic.io/ionicons
I did't added any autotest. It's already well tested feature and I only extend it by adding new usage options.
Also there is no need in adding translation here, because notepad icon (old version before symbol api) was already translated on sth else.

Connected PR's:
https://github.com/jenkinsci/workflow-job-plugin/pull/274

### Proposed changelog entries

* -

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@NotMyFault @janfaracik @uhafner 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6800"></a>

### CODE REVIEW

**Review by NotMyFault — COMMENTED** [review]
The `open-note` one looks a bit out of style to me, looks a bit like a hyperlink to a third party website or something.

File: core/src/main/resources/hudson/model/AbstractBuild/tasks.jelly
Comment (timja) [inline core/src/main/resources/hudson/model/AbstractBuild/tasks.jelly:39]:
```suggestion
              <l:task icon="symbol-open-note" href="${buildUrl.baseUrl}/configure" title="${%View Build Information}"/>
```

**Review by timja — APPROVED** [review]

**Comment by DuMaM** [comment]
> The `open-note` one looks a bit out of style to me, looks a bit like a hyperlink to a third party website or something.

Any suggestions? So I can also update this one? 😉 
I can edit it a bit and add an eye/glasses instead of arrow.
glasses-outline
eye-outline

https://github.com/jenkinsci/workflow-job-plugin/pull/274

**Comment by NotMyFault** [comment]
The eye looks fine, it's already used in other occasions to demonstrate something an only be looked at.

**Comment by DuMaM** [comment]
> The eye looks fine, it's already used in other occasions to demonstrate something an only be looked at, we can reuse it here too.

I can merge eye and pencil with this: https://github.com/jenkinsci/jenkins/blob/master/war/src/main/resources/images/symbols/details.svg

To have similar effect to this one
https://icon-library.com/icon/show-all-icon-1.html

Or maybe we should keep it simple use eye only with this edit-note

**Comment by NotMyFault** [comment]
> > The eye looks fine, it's already used in other occasions to demonstrate something an only be looked at, we can reuse it here too.
> 
> I can merge eye and pencil with this: https://github.com/jenkinsci/jenkins/blob/master/war/src/main/resources/images/symbols/details.svg
> 
> To have similar effect to this one https://icon-library.com/icon/show-all-icon-1.html At the end i will have a edit-doc and view-doc symbols
> 
> Or maybe we should keep it simple and use eye only instead of edit-note

The eye itself is fine imo

**Review by NotMyFault — APPROVED** [review]

**Review by yaroslavafenkin — APPROVED** [review]

**Comment by NotMyFault** [comment]
This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Comment by DuMaM** [comment]
do we merge it?

**Review by basil — APPROVED** [review]
Thanks for the PR!

**Pull request merged**
